lib/util.js

```diff
@@ -136,6 +137,28 @@ util = {
 
             callback(null, data);
         });
+    },
+
+    /**
+     * Writes a JSON blob to the given path.
+     *  * If ``options.pretty`` is true, output will be pretty printed. (Default false)
+     *  * If ``options.overwrite`` is false, output file will be overwritten (Default true)
+     *
+     * @param path
+     * @param data
+     * @param options
+     * @param callback
+     */
+    writeJSON: function (path, data, options, callback) {
+        var json = (options.pretty) ? JSON.stringify(data, null, 4) : JSON.stringify(data),
+            flag = (options.overwrite) ? 'w' : 'wx';
+
+        fs.writeFile(path, json, {flag: flag}, function (err) {
+            if (err) {
+                return callback(err);
+            }
+            callback();
```

Comment:
This if-block and the entire function looks redundant, just passing the `callback` to `fs.writeFile` would do the same thing.